### DISCUSSION TITLE
Add activity-stream to API

### DISCUSSION THREAD

**Discussion by easysugar** [crowdin/crowdin-api-client-python#37] [Q&A]
Hi. I see that backend at Crowdin has its own endpoint for getting the project's last activity: `/backend/project_actions/activity_stream`. 
But I cannot find where this method is in the client API. So can you add this endpoint to the client, please?

**Comment by andrii-bodnar** [comment]
Hi @easysugar!

The Crowdin API Client Python - is an interface to the public APIs - [crowdin.com](https://developer.crowdin.com/api/v2/), [Crowdin Enterprise](https://developer.crowdin.com/enterprise/api/v2/). It's not supposed to support these backend endpoints.

**Comment by easysugar** [comment]
@andrii-bodnar 
OK. But are you planning to add the same endpoint for the last activity to the public API?

**Comment by andrii-bodnar** [comment]
@easysugar currently, we are not planning such an endpoint in the nearest future, but I need to discuss it with the Team.

Could you please describe your use case in more detail? It will help us a lot with designing this new Activity API. Or we can find a workaround for this case.

**Comment by easysugar** [comment]
Sure. I want to calculate something like "top contributors" for last week using the last activity.
Currently, I have to use different endpoints (for translations, approvals, comments), get all historical data, and then filter by date.

**Comment by andrii-bodnar** [comment]
@easysugar please check the [Reports](https://developer.crowdin.com/api/v2/#tag/Reports) section. 

The **Top Members** or **Contribution Raw Data** report should help to collect data regarding users' contributions.

**Comment by easysugar** [comment]
Actually, I would like to make a report by myself. But I will try Reports.

**Comment by andrii-bodnar** [comment]
The **Contribution Raw Data** report contains just the raw data regarding contributions so you will be able to calculate the report you want based on this data.

Also, you can check the [Raw Report Data app](https://store.crowdin.com/raw-report). This app is using the same API call to generate the report.

**Reply by easysugar** [reply]
@andrii-bodnar found that reports can be consumed only by excel format, which isn't comfortable to use in Python. 
Also, reports don't contain information about comments count, what I need to use.
